benchmarks/fogbench/__main__.py

```diff
@@ -184,6 +208,30 @@ def read_out_file(_file=None, _keep=False):
     print('Result: %s\n' % response.code)
 
 
+def display_statistics(stats_type):
+    print(u"{} statistics: ".format(stats_type))
+    global _START_TIME
+    global _END_TIME
+    global _TOT_MSGS_TRANSFERRED
+    global _TOT_BYTE_TRANSFERRED
+    global _NUM_ITERATED
+    if stats_type == 'total' or stats_type == 'st':
+        print(u"Start Time::{}".format(_START_TIME))
```

Comment:
for start and end time; could you please also show 
`print(u"Start Time::{} | {}".format(_START_TIME, datetime.now()))` 
> Start Time::2017-08-08 12:24:27.601015+00:00 | 2017-08-08 17:55:22.261695

This will help to read stats quickly